mappings/lowfat-macula-hebrew.xquery

```diff
@@ -194,7 +194,10 @@ declare function local:oneword($node as element(Node))
             if ($node/c)
             then
                 ()
-            else $node
```

Comment:
In returning the `Node` directly here, it would not process the node as a word under specific circumstances.